dataactbroker/README.md

```diff
@@ -721,6 +721,34 @@ This route certifies the specified submission, if possible. If a submission has
 ```
 * `message` - A message indicating whether or not the action was successful. Any message other than "Success" indicates a failure.
 
+#### GET "/v1/gtas_window"
+
+This route checks if there is a gtas window currently open, and if it is returns the start and end date, else returns None
+
+##### Body (JSON)
+
+```
+{}
+```
+
+##### Body Description
+
+
+##### Response (JSON)
+
+```
+{}
```

Comment:
This is not the response. The response you show here is just an example so you don't have to show an "or". Also, the response is
```
{
  data: {
       start_date: '2012-05-17',
       end_date: '2012-06-17'
   }
}
```